src/http/request.cr

```diff
@@ -26,7 +27,7 @@ class HTTP::Request
 
   def to_io(io)
     io << @method << " " << @path << " " << @version << "\r\n"
-    HTTP.serialize_headers_and_body(io, @headers, @body)
+    HTTP.serialize_headers_and_body(io, @headers, @cookies, @body, true)
```

Comment:
How about moving the cookie header generation into `Cookies#add_response_headers` so this becomes `HTTP.serialize_headers_and_body(io, @cookies.add_response_headers(@headers), @body)` or something like that?